components/project/sections/TransactionsSection.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <n-card class="w-full" title="Transactions">
+    <template #header-extra>
+      <n-button type="primary" text @click="refreshTransactions()">
+        <Icon size="25" name="material-symbols:sync-rounded" />
+      </n-button>
+    </template>
+    <n-space vertical>
+      <n-input v-model:value="searchValues.senderId" type="text" placeholder="Sender ID" />
+      <n-input v-model:value="searchValues.receiverId" type="text" placeholder="Receiver ID" />
+      <n-input v-model:value="searchValues.nftId" type="text" placeholder="NFT ID" />
+      <n-button @click="handleSearch()">Search</n-button>
+      <n-data-table
+        v-if="!error"
+        ref="table"
+        remote
+        :columns="columns"
+        :data="transactionsResponse?.transactions"
+        :loading="transactionsLoading"
+        :pagination="pagination"
+      />
+      <div v-else>Error: {{ error }}</div>
+    </n-space>
+  </n-card>
+</template>
+
+<script setup lang="ts">
+import { NDataTable, NCard, NButton, NSpace, NInput } from 'naive-ui';
```

Comment:
You dont have to do this in this task, but we should switch to nuxt module naiveui. There is much less config required and all the naive components are autoinjected. I tested it and it works like charm